aleph-client/Cargo.toml

```diff
@@ -1,28 +1,31 @@
 [package]
 name = "aleph_client"
-version = "0.4.2"
+version = "0.4.0"
 edition = "2021"
 
+# client
+[dependencies.substrate-api-client]
```

Comment:
matter of taste, but maybe let's be consistent and inline this dependency like:
`substrate-api-client = { git = ..., branch = ..., features = ... }`